Dockerfile

```diff
@@ -144,19 +130,12 @@ RUN pip install --no-deps --root=/install-root .
 
 #######################################################################
 
-# Separate stage to build the C++ tools. This is in a separate build stage
-# so that changes to either the C++ code or the Python code don't invalidate
-# the build cache for the other.
+# Separate stage to build C tools. This is in a separate build stage
+# so that changes to the package don't invalidate the build cache for this.
 
-FROM build-base as build-cxx
+FROM build-base as build-c
 
-# Build utilities.
-# We use make clean to ensure that an existing build from the build context
-# won't accidentally get used instead.
 WORKDIR /tmp/tools
-COPY src/tools .
-RUN make clean && make -j fsim
-
 RUN wget https://raw.githubusercontent.com/ska-sa/katsdpdockerbase/master/docker-base-runtime/schedrr.c && \
```

Comment:
It runs a child process with real-time scheduling requested, provided that schedrr itself has the required capabilities. So a bit like `spead2_net_raw` but for real-time scheduling instead of raw network device access.

`chrt` would do the same thing, but it seems to launch the target as a child process and keeps running rather than using `exec` to replace itself, which causes problems when you want to send signals and the like.